docs/deployment.md

```diff
@@ -1,17 +1,15 @@
 # Deployment
 
-## Before you merge
-
 This app only has minimal CI set up, running a single check to ensure that Smokey is able to make requests. It would be misleading to run all of the tests on CI because their behaviour changes depending on the environment they are run in. You should manually test in applicable environments, until you are confident your change is not a breaking one.
 
 Example steps to test in Integration:
 
-- Click "Configure" for the [Smokey job][]
-- Change the "Branch specifier" to the name of your branch
-- Run a new build of the Smokey project and check the results
-
-## After you merge
+- Navigate to the [Actions](https://github.com/alphagov/smokey/actions) tab on the repo
```

Comment:
```suggestion
- Navigate to the [Deploy GitHub Action](https://github.com/alphagov/smokey/actions/workflows/deploy.yml) tab on the repo
```

Reply:
You need to be on the specific workflow before you get given the "Run workflow" button.